### DISCUSSION TITLE
Deploying to a local blockchain (Scripting)

### DISCUSSION THREAD

**Discussion by QsAdedipe** [Cyfrin/foundry-full-course-cu#1529] [Q&A] [answered]
while trying import forge-std file, i discovered the lid folder is empty and does not have any file
this is the line of code but I 
can not deploy because the forge-std folder does not exist

'''
//SPDX-License-Identifier: MIT
pragma solidity ^0.8.19;

import "forge-std/Script.sol";

import{SimpleStorage} from "../src/SimpleStorage.sol" ;

contract DeploySimpleStorage is Script{    
}

'''

**Comment by EngrPips** [comment, marked as answer]
Hello @QsAdedipe , please run `forge install foundry-rs/forge-std --no-commit` in your command line and the library should be imported into your project

**Comment by QsAdedipe** [comment]
Thank you for your response, I really appreciate it.
 It has been imported

**Reply by EngrPips** [reply]
Awesome, glad you were able to fix the issue. You can as well mark this conversation as answered

**Comment by Bloodday** [comment]
I had the same problem but i ran foundry inir --force and I got the files, but now I can't import "foundry-std/xxxx" I have to be more specific on the path, like "../lib/foundry-std/src/xxxx". 

is there a way to fix this? is a problem of the vs code extension that highlight/lint the code?

**Reply by EngrPips** [reply]
Hello @Bloodday , you won't be able to import `foundry-std/xxxx` but you should be able to import `forge-std/xxxx`. Thats because there is no folder as `foundry-std` in your project library

**Reply by Bloodday** [reply]
your are right, is forge-std, but it still mark it as an error (it still compiles tho)

**Reply by EngrPips** [reply]
Check and confirm that your path is correct, you should be able to import the file
